pkg/logs/agent.go

```diff
@@ -104,11 +106,12 @@ func NewServerless(sources *config.LogSources, services *service.Services, proce
 	}
 
 	return &Agent{
-		auditor:          auditor,
-		destinationsCtx:  destinationsCtx,
-		pipelineProvider: pipelineProvider,
-		inputs:           inputs,
-		health:           health,
+		auditor:                   auditor,
+		destinationsCtx:           destinationsCtx,
+		pipelineProvider:          pipelineProvider,
+		inputs:                    inputs,
+		health:                    health,
+		diagnosticMessageReceiver: diagnosticMessageReceiver,
```

Comment:
@remeh -- I added this because without it, the logs agent was crashing

Reply:
Ok! That was the right thing to do, this is something brand new (21st of January) in `master` which is indeed needed!